tests/integration/identity.go

```diff
@@ -208,3 +208,9 @@ func getIdentityDID(s *state, identity immutable.Option[Identity]) string {
 	}
 	return ""
 }
+
+// resetContextWithNoIdentity resets identity for the ctx to avoid, leaving it there and having the ctx
+// reuse the same identity for other requests that don't specify an identity.
+func resetStateContext(s *state) {
+	s.ctx = acpIdentity.WithContext(s.ctx, acpIdentity.None)
+}
```

Comment:
thought: This function illustrate the problem with using context for required parameters between packages.